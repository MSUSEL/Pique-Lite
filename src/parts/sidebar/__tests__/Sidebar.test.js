import React from 'react';
import {shallow} from 'enzyme';

import Sidebar from '../Sidebar.component';

describe('Sidebar Component', () => {
    let wrapper;
    wrapper = shallow(<Sidebar/>).dive();

    it('should render Sidebar component', () => {
        expect(wrapper).toMatchSnapshot();
    })
})