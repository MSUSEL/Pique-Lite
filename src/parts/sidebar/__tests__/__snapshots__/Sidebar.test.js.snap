// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar Component should render Sidebar component 1`] = `ShallowWrapper {}`;
